registry/network/helpers.go

```diff
@@ -0,0 +1,50 @@
+package network
+
+import (
+	"net"
+)
+
+// determineIPVersion determines the IP version (IPv4 or IPv6) for a given net.IP.
+//
+// It checks the length of the IP address to identify the IP version:
+// - Returns 4 if it's an IPv4 address.
+// - Returns 6 if it's an IPv6 address.
+//
+// Parameters:
+//
+//	ip net.IP - the IP address to check.
+//
+// Returns:
+//
+//	int - 4 for IPv4, 6 for IPv6.
+func (nr *NetworkRegistry) determineIPVersion(ip net.IP) int {
```

Comment:
I know memory cost almost nothing, but returning a uint8 would be enough?

simply a string or maybe a byte would be simpler no? I mean at the end, everything becomes a string with a templating system

Reply:
Yes uint8 are enough !